introduction.md

```diff
@@ -6,27 +6,120 @@ title: Introduction
+* **Create cloud instances**: Integrate with clouds like OpenStack, Rackspace, Amazon EC2 or Google Compute Engine directly from Foreman UI/API/CLI.
+* **Host network configuration**: Provisioning templates which create network configuration for installed hosts including bonding, bridging and VLAN trunk support.
+* **Configuration management bootstrap**: Template snippets for bootstrapping initial configuration of configuration management software including signing client keys with CA.
+* **IPv6**: Foreman can manage IPv6 addresses on non-provisioning interfaces (PXE provisioning on IPv6 is [work in progress](link to redmine bug goes here))
+* **Templating engine**: Templates based on ERB for OS installation recipes (Kickstart, Preseed), jobs (SSH scripts, Ansible jobs), partitioning schemes and other types.
+* **Compute Resources**: Modules or plugins for integration with hypervisors and cloud infrastructure.
+* **Compute Profiles**: Common compute profiles across multiple clouds or virtualization (e.g. xxsmall, large, medium).
+
+**Server discovery**
+
+* **Host discovery**: Boot unknown hardware from network or via local media (USB stick) and let it register to Foreman for automatic, semi-automatic or fully manual provisioning.
+* **Provisioning of discovered nodes**: Automatic, semi-automatic or fully manual provisioning of discovered hardware via WebUI/CLI/API.
+
+**Large teams support**
+
+* **Host parameters**: Flexible parameters engine for hosts and associated objects (subnets, domains, host groups) with dynamically generated ones called Smart Variables/Class Parameters.
+* **Foreman proxies**: Components running inside data centres, subnets or remote sites providing connection to managed nodes and services using REST HTTPS API.
+* **Authentication**: Username and password authentication with brute-force protection, POSIX LDAP, FreeIPA and MSAD authentication integration.
+* **Authorization**: Fine-grained role-based access controls (RBAC) for users, roles, LDAP mapping
+* **Authorization filters**: Ability to assign authorization permissions to filtered objects (e.g. hostnames starting with 'test-').
+* **Multitenancy**: Most resources in Foreman can be assigned to Organizations and Locations as a flexible authorization mechanism for multiple organizations or sites.
+* **Kerberos**: Foreman supports automatically creating FreeIPA Realm entries for new hosts.
+* **HTTP Proxy**: For some communication of managed nodes or Foreman itself.
+
+**Reporting and monitoring**
+
+* **Dashboard**: Fully configurable dashboard with widgets and statistics.
+* **Facts**: Inventory of facts reported by configuration management agents (Facter, Ansible, Salt grains).
+* **Trends**: Track changes in Foreman infrastructure over time, including key Foreman resources or facts.
+* **Audit**: Detailed audit trail with per-field granularity and `diff` feature for templates.
```

Comment:
maybe instead of just templates, config changes? (such as templates, configuration management reports etc)

Reply:
OK